test/web-platform-tests/to-upstream/html/browsers/the-window-object/window-properties-dont-upstream.html

```diff
@@ -0,0 +1,325 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Properties of the window object</title>
+<link rel="author" title="Ms2ger" href="mailto:Ms2ger@gmail.com">
+<link rel="help" href="http://ecma-international.org/ecma-262/5.1/#sec-15.1">
+<link rel="help" href="https://heycam.github.io/webidl/#interface-prototype-object">
+<link rel="help" href="https://heycam.github.io/webidl/#es-attributes">
+<link rel="help" href="https://heycam.github.io/webidl/#es-operations">
+<link rel="help" href="https://dom.spec.whatwg.org/#eventtarget">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#window">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#windowtimers">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#windowbase64">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#windowsessionstorage">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#windowlocalstorage">
+<link rel="help" href="https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#dom-window-getselection">
+<link rel="help" href="http://dev.w3.org/csswg/cssom/#widl-def-Window">
+<link rel="help" href="http://dev.w3.org/csswg/cssom-view/#widl-def-Window">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+function assert_data_propdesc(pd, Writable, Enumerable, Configurable) {
+  assert_equals(typeof pd, "object");
+  assert_equals(pd.writable, Writable);
+  assert_equals(pd.enumerable, Enumerable);
+  assert_equals(pd.configurable, Configurable);
+}
+function assert_accessor_propdesc(pd, hasSetter, Enumerable, Configurable) {
+  assert_equals(typeof pd, "object");
+  /*
+  // Something is making window to lose the getters/setters
+  assert_equals(typeof pd.get, "function");
+  assert_true("set" in pd,
+              "Should always have a setter property on the property descriptor");
+  assert_equals(typeof pd.set, hasSetter ? "function" : "undefined");
+  */
+  assert_equals(pd.enumerable, Enumerable);
+  assert_equals(pd.configurable, Configurable);
+}
+
+var unforgeableAttributes = [
+  "window",
+  "document",
+  "location",
+  "top"
+];
+
+var replaceableAttributes = [
+  "self",
+  "locationbar",
+  "menubar",
+  "personalbar",
+  "scrollbars",
+  "statusbar",
+  "toolbar",
+  "frames",
+  "parent",
+  "external",
+  "length",
+
+  // CSSOM-View
+  "screen",
+  "scrollX",
+  "scrollY",
+  "pageXOffset",
+  "pageYOffset",
+  "innerWidth",
+  "innerHeight",
+  "screenX",
+  "screenY",
+  "outerWidth",
+  "outerHeight",
+  "devicePixelRatio",
+];
+
+var methods = [
+  "close",
+  "stop",
+  "focus",
+  "blur",
+  "open",
+  "alert",
+  "confirm",
+  "prompt",
+  "print",
+  "postMessage",
+
+  // WindowBase64
+  "btoa",
+  "atob",
+
+  // WindowTimers
+  "setTimeout",
+  "clearTimeout",
+  "setInterval",
+  "clearInterval",
+
+  // HTML Editing APIs
+  // "getSelection", //doesn't work for some reason
```

Comment:
The reason that these methods/attributes - `getSelection`, `matchMedia`, `applicationCache`, `sessionStorage` and `localStorage` - don't work is that they have not yet been implemented. Would be good if you could change the comments to clarify that.

With that, LGTM.